src/scripts/CompleteButton.js

```diff
@@ -19,16 +17,12 @@ const onSubmitRequestPressed = (event) =>
     }
     */
   }
-}
+};
 
-export const CompleteButton = () =>
-{
-  document.addEventListener(
-    "mouseup",
-    onSubmitRequestPressed
-  )
+export const CompleteButton = () => {
+  document.addEventListener("mouseup", onSubmitRequestPressed);
 
-  let returnHTML = `<button class="complete-button__button" id="submit-request">Submit Request</button>`
+  let returnHTML = `<button class="complete-button__button formSection" id="submit-request">Submit Request</button>`;
```

Comment:
The added formSection class is messing with formatting

Reply:
Prettier Extension.